README.md

```diff
@@ -1,2 +1,10 @@
-# hw01
-Repository for homework 01
+# Abby Bergman ![image](https://github.com/abbygirlrose/hw01/blob/master/ab.png)
+
+## Background
+#### I am from Los Angeles, California and have spent four years living in Western Massachusetts. I have a sister and a cat. 
+
+## Education
+#### I am a MAPSS student concentrating in **Psychology**. My research interests center around *motivation*, *decision making*, and *goals*. I completed my undergraduate education in **Neuroscience** and **Exercise and Sports Studies** at [Smith College](http://www.smith.edu), where I also competed on the varsity swim team. 
+
+## Interests
+#### I am fairly obsessed with swimming and will be volunteer coaching here at U of C. I am a marathon swimmer and details of some of my swims are located [here](http://abbybergman/com). 
```

Comment:
I think the error in this link is that it's a backslash and not a period before 'com'. Might be a good habit to check the links in rich text before submitting, just in case.